File: src/Java/Odd Echo/OddEcho.java
import java.util.Scanner;

public class OddEcho {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    int n = sc.nextInt();
    for (int i = 0; i < n; i++) {
      String word = sc.next();
      if (i % 2 == 0) {
        System.out.println(word);
      }
    }
    sc.close();
  }
}

File: src/C++/Odd Echo/OddEcho.cpp
#include <iostream>

int main() {
  int n ;
  std::cin >> n;
  for (int i = 0; i < n; i++) {
    std::string word;
    std::cin >> word;
    if (i % 2 == 0) {
      std::cout << word << "\n";
    }
  }
  return 0;
}